### ISSUE TITLE
space-after-keywords replaced with keyword-spacing in eslint@2

### ISSUE THREAD

**Issue by joelmukuthu** [One-com/eslint-config-onelint#12]
eslint 2 complains about this rule:

```
Rule 'space-after-keywords' was removed and replaced by: keyword-spacing
```

and it's actually being thrown as a lint error and not a warning or something :/

**Comment by joelmukuthu** [comment]
I could have changed it and created a PR but I'm not sure what other projects are relying on 'space-after-keywords' i.e. using eslint@1

**Comment by gustavnikolaj** [comment]
We should definitely fix this, but I'd prefer to finish the port of onelint from the current jshint/jscs setup first, before introducing a release branch. Could you downgrade to eslint ^1 for now? I'll make sure to fix this as soon as I've got time :-)

**Comment by joelmukuthu** [comment]
Sure :)

**Comment by gustavnikolaj** [comment]
fixed in https://github.com/One-com/eslint-config-onelint/commit/f4a3e5a21e98d3c3459d9c208eece4eab8f045af and will be released as v1.0.0 :-)
